state/unit_ops.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+import (
+	"github.com/juju/errors"
+	mgoutils "github.com/juju/juju/mongo/utils"
```

Comment:
this import should be by itself at the bottom

Reply:
Resolved in https://github.com/juju/juju/pull/11204/commits/5517d74c8c0525437ee86833c30c4c326603f217